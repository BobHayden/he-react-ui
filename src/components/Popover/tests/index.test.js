import createTestContext from "react-cosmos-test/enzyme";
import fixture from "../fixtures/Default.fixture";

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

test("<Popover /> rendered correctly with fixture", () => {
  const wrapper = getWrapper();
  expect(wrapper.find('div[role="tooltip"]').length).toEqual(1);
  expect(wrapper.find('div[role="tooltip"] div').length).toEqual(3);

  expect(wrapper).toMatchSnapshot();
});
