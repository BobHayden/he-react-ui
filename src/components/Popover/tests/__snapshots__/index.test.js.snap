// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Popover /> rendered correctly with fixture 1`] = `
<Popover
  placement="bottom"
  positionLeft={200}
  positionTop={50}
>
  <div
    className="undefined"
    role="tooltip"
    style={
      Object {
        "display": "block",
        "left": 200,
        "top": 50,
      }
    }
  >
    <div
      style={
        Object {
          "left": undefined,
          "top": undefined,
        }
      }
    />
    <div>
      The patient associated with this recall does not have a valid Australian mobile number.
    </div>
  </div>
</Popover>
`;
